src/components/SearchComponents/Search.page.test.tsx

```diff
@@ -297,4 +368,61 @@ describe("Search.page.tsx", () => {
 
     expect(setSearchStartTime).toHaveBeenCalledTimes(1);
   });
+  test("searchResultClicked is called when a search hit is clicked", async () => {
+    const { getByText } = render(<Search {...props} {...resultsProps} />);
+    const description = getByText("lesson title");
+    const user = userEvent.setup();
+    await act(async () => {
```

Comment:
I don't think you need to wrap this in an act, you should be able to just await the user.click

Reply:
I added this to suppress the waring in the test 
```
 Warning: An update to ForwardRef(LinkComponent) inside a test was not wrapped in act(...).
    
    When testing, code that causes React state updates should be wrapped into act(...):
    
    act(() => {
      /* fire events that update state */
    });
    /* assert on the output */
```

Adding the act removes the warning